client/plots/geneExpression.js

```diff
@@ -3,7 +3,8 @@ import { dofetch3 } from '#common/dofetch'
 import { interpolateRgb } from 'd3-interpolate'
```

Comment:
looks like `geneExpression.js` is not used right now so no need to modify it

Reply:
As we discussed, we will keep this for consistency so that it can be used later.